Demo/Components/ObjectPageTitleView/ObjectPageTitleDemoView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+class ObjectPageTitleDemoView: UIView, Tweakable {
+
+    private var titleView: ObjectPageTitleView?
+
+    lazy var tweakingOptions: [TweakingOption] = {
+        [
+            TweakingOption(title: "Title and subtitle", description: "Default style") { [weak self] in
+                self?.setupTitleView(title: "Mercedes-Benz C-Klasse", subtitle: "C200 4MATIC aut Hengerfeste, Panoramasoltak, AMG, LED +")
+            },
+            TweakingOption(title: "Only title", description: "Default style") { [weak self] in
+                self?.setupTitleView(title: "Sofa med sjeselong - pris diskuterbar!")
+            },
+            TweakingOption(title: "Only subtitle", description: "Default style") { [weak self] in
+                self?.setupTitleView(subtitle: "C200 4MATIC aut Hengerfeste, Panoramasoltak, AMG, LED +")
+            },
+        ]
+    }()
+
+    // MARK: - Init
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        tweakingOptions.first?.action?()
+    }
+
+    public required init?(coder aDecoder: NSCoder) { fatalError() }
+
+    // MARK: - Private methods
+
+    private func setupTitleView(
+        title: String? = nil,
+        subtitle: String? = nil,
+        titleStyle: Label.Style = .title2,
+        subtitleStyle: Label.Style = .body
+    ) {
+        titleView?.removeFromSuperview()
+        titleView = nil
+
+        let newTitleView = ObjectPageTitleView(titleStyle: titleStyle, subtitleStyle: subtitleStyle, withAutoLayout: true)
+        addSubview(newTitleView)
+
+        newTitleView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: .mediumSpacing).isActive = true
+        newTitleView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -.mediumSpacing).isActive = true
+        newTitleView.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
+        newTitleView.configure(withTitle: title, subtitle: subtitle)
+        self.titleView = newTitleView
```

Comment:
just nitpicking
if you have multiple constraints i think it's nicer to write

```swift
NSLayoutConstraint.activate([
])
```

instead of multiple `isActive = true`.

Also, why do you need to create a new title view each time? can you use the instance and just call the `configure` method within the tweaking actions?

Reply:
I'll fix the constraints 👍

I did this, because I was unsure if we needed to simulate other markets in the demo, since some of those will have different needs in regard to label styling. We don't have this issue right now, so I'll just keep the one instance instead of recreating the view.